awsf3/cloudwatch_agent_config.json

```diff
@@ -19,7 +19,8 @@
 				"measurement": [
 					"usage_idle",
 					"usage_iowait",
-					"usage_system"
+					"usage_system",
```

Comment:
Might consider eliminating this altogether since we aren't using it and don't care about it (I don't think?)

Reply:
Sounds good